newIDE/app/src/MainFrame/EditorContainers/SceneEditorContainer.js

```diff
@@ -54,6 +55,31 @@ export class SceneEditorContainer extends React.Component<RenderEditorContainerP
     }
   }
 
+  onEventsBasedObjectChildrenEdited() {
+    const { editor } = this;
+    if (editor) {
+      // Update every custom objects because some custom objects may include
+      // the one actually edited.
+      editor.forceUpdateCustomObjectRenderedInstances();
+    }
+  }
+
+  onSceneObjectEdited(scene: gdLayout, objectWithContext: ObjectWithContext) {
+    const layout = this.getLayout();
+    if (!layout) {
+      return;
+    }
+    if (layout !== scene && !objectWithContext.global) {
```

Comment:
Should that be:
```suggestion
    if (layout === scene || !objectWithContext.global) {
```
?